packages/app/src/app/components/WorkspaceSetup/steps/SelectWorkspace.tsx

```diff
@@ -84,6 +84,7 @@ export const SelectWorkspace: React.FC<StepProps> = ({
             name="workspace"
             label="Select the workspace you want to upgrade"
             onChange={ev => setWorkspaceId(ev.target.value)}
+            value={workspaceId}
```

Comment:
Here's the original problem: as no initial workspace ID was passed, the select showed the first item as selected, although it could be any option.

Reply:
nice catch!